actions.md

```diff
@@ -0,0 +1,547 @@
+
+  **[[NH: please check command and output:]]**
+  ```
+  nim action get /whisk-system/samples/greeting --save
+  ok: saved action code to /path/to/namespace/greeting.js
+  ```
+
+2. You can provide your own file name and extension by using the `--save-as` flag.
+
+  **[[NH: check output:]]**
+  ```
+  wsk action get /whisk-system/samples/greeting --save-as hello.js
+  ok: saved action code to /path/to/namespace/hello.js
+  ```
+
+### List actions
+
+You can list all the actions that you have created using the following commnand:
+
+```
+nim action list
+  actions
+  /guest/mySequence                  private sequence
+  /guest/greeting                    private nodejs:6
+```
+
+Here, we see actions listed in order from most to least recently updated. For easier browsing, you can use the flag `--name-sort` or `-n` to sort the list alphabetically:
+
+**[[NH: Output list seems backwards for alphabetical order?]]**
+```
+nim action list --name-sort
+  actions
+  /guest/mySequence                  private sequence
+  /guest/greeting                    private nodejs:6
+```
+
+The list is now sorted alphabetically by namespace, then package name if any, and finally action name, with the default package (no specified package) listed at the top. **[[NH: Is namespace listed in this output example? What does output look like if there's a package? Does nim output list the namespace?]]**
+
+**Note**: The printed list is sorted alphabetically after it is received from the platform. Other list flags such as `--limit` and `--skip` are applied to the block of actions before they are received for sorting. To list actions in order by creation time, use the flag `--time`.
+
+To filter your list of actions to just those within a specific package, use this command:
+
+**[[NH: please check command and output:]]**
+```
+him action list /whisk-system/utils
+```
+```
+actions
+/whisk-system/utils/hosturl        private nodejs:6
+/whisk-system/utils/namespace      private nodejs:6
+/whisk-system/utils/cat            private nodejs:6
+/whisk-system/utils/smash          private nodejs:6
+/whisk-system/utils/echo           private nodejs:6
+/whisk-system/utils/split          private nodejs:6
+/whisk-system/utils/date           private nodejs:6
+/whisk-system/utils/head           private nodejs:6
+/whisk-system/utils/sort           private nodejs:6
+```
+
+## Delete actions
+
+You can clean up by deleting actions that you do not want to use.
+
+1. Run the following command to delete an action:
+
+  ```
+  nim action delete greeting
+  ok: deleted greeting
+  ```
+
+2. Verify that the action no longer appears in the list of actions.
+
+**[[NH: please check output:]]**
+  ```
+  nim action list
+  ```
+  ```
+  actions
+  /guest/mySequence                private sequence
+
+  ```
+
+## Access action metadata within the action body
+
+**[[NH: "OpenWhisk assets" in next paragraph ok?]]**
```

Comment:
OpenWhisk in general is ok.  The Nimbella information model is a superset of the Apache OpenWhisk information model, a fact that we sometimes even highlight.